app/pages/NavigatorDashboard/NavigatorDashboard.tsx

```diff
@@ -94,73 +144,152 @@ const AdvancedSearch = () => {
   };
 
   return (
-    <div className={styles.advancedSearchContainer}>
-      <h1 className={styles.header}>Hi, Navigator :)</h1>
-      <p className={styles.blurb}>
-        Please use the search bar to find resources for the individuals
-        you&apos;re assisting. You can enter keywords such as
-        &ldquo;shelter,&ldquo; &ldquo;food assistance,&ldquo; or
-        &ldquo;employment support,&ldquo; to discover relevant programs and
-        services.
-      </p>
-      <form
-        className={styles.searchBoxForm}
-        onSubmit={(evt) => {
-          evt.preventDefault();
-          submitSearch();
-        }}
-      >
-        <div className={styles.searchInputContainer}>
-          <input
-            type="text"
-            placeholder="Emergency Shelter"
-            className={styles.searchInput}
-            onChange={(evt) => setSearchValue(evt.target.value)}
-          />
-          <Button
-            buttonType="submit"
-            // Rather using the disabled attr when advanced search mode is active, we pseudo-disable
-            // the button by adding styling. This is because using the disabled attr breaks using the
-            // "enter" button to submit the form when focused on the search input
-            addClass={`${styles.searchButton} ${
-              expandedOptions ? styles.disabled : ""
-            }`}
-          >
-            Search
-          </Button>
-          <button
-            type="button"
-            className={`${styles.searchFilterButton} ${
-              expandedOptions ? styles.expandedOptions : ""
-            }`}
-            onClick={() => setExpandedOptions(!expandedOptions)}
-          >
-            <img
-              src={icon(expandedOptions ? "filter-white" : "filter-gray")}
-              alt="Add filters to search"
+    <>
+      <div className={styles.advancedSearchContainer}>
```

Comment:
I had to create a parent tag and nest the advancedSearchContainer element to include the Bookmark/Saved Search UI on the page. In other words, there are no substantive changes from lines 148–215; the only changes are tabbing to nest the element